packages/models/proto/v2/authorization/key.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package authorization.v2;
+
+message KeyEntryV2 {
+  string keyId = 1;
+  KeyV2 value = 2;
+}
```

Comment:
as you correctly said, this can me removed
```suggestion
```

Reply:
Removed